packages/experimental/src/components/APIKeyModal/_api-key-modal.scss

```diff
@@ -32,3 +28,7 @@
 .#{$exp-prefix}--apikey-modal-messaging-text > * {
   font: inherit;
 }
+
+.#{$exp-prefix}--apikey-modal .bx--modal-content {
+  padding-right: $spacing-05;
```

Comment:
Can we use carbon tokens here from there guidelines, rather than hardcoded values?

Reply:
i don't understand. this is a carbon token.

Reply:
ups, wrong line, I'll re-review now 👍